docs/blog.html

```diff
@@ -0,0 +1,205 @@
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Introduction to the Administration Portal</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Introduction to the VM Portal</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Self-Hosted Engine Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Data Warehouse Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Metrics Store Installation Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Metrics Store User Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Disaster Recovery Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Gluster Hyperconvergence Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Orb</a></li>
+        <li class="ov-nav-main__sub-item ov-nav-main__sub-item--heading">Release notes</li>
+        <li class="ov-nav-main__sub-item">Latest Release Notes</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Data Warehouse Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Metrics Store Installation Guide</a></li>
+        <li class="ov-nav-main__sub-item">Older Release Notes</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Data Warehouse Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Metrics Store Installation Guide</a></li>
+      </ul>
+    </li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="developers.html">Developers</a>
+    </li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="community.html">Community</a></li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="https://lists.ovirt.org/archives/">Forum</a></li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="blog.html">Blog</a></li>
+  </ul>
+</nav>
+  </header>
+  
+
+  <main class="ov-main ov-main--home">
+    <section class="ov-masthead">
+    <section class="ov-masthead--blog">  
+      <h1 class="ov-heading ov-heading--xl">oVirt Blog</h1>
+      <h2 class="ov-heading ov-heading--lg">Posts from the community and beyond.</h2>
+      <a href="" class="ov-button ov-button--primary">Contribute</a>
+      </section>
+    </section>
+  
+    
+    <section class="ov-site-section ov-posts ov-posts--home">
```

Comment:
Talking more about grid would be great and using this as an example would be really helpful since I don't have too much familiarity with some of the workings of the grid.